### PR TITLE
Add docs/ submodule and supporting twig structure.

### PR DESCRIPTION
**Pull request by kwhitley**
**TODO**
Path cleanup on aisle 3?

Refs #22

### CODE REVIEW

File: src/client/less/app.less [hunk 1]
```diff
@@ -57,6 +57,15 @@
   }
 }
 
+#content {
```
Comment (ericclemmons) [inline src/client/less/app.less:60]: I assume this was necessary for subpages?

**Comment by ericclemmons** [comment]
This layout looks very smart.  I'm going to run this locally to get a better gist of how it works together.

I also want to remove the usage of `package.json` within the project.  It was neat to test with, but in actuality, _no one_ would actually build their app to use that :)  Also, it would get rid of the "wth" factor when looking over the code, and just be one more thing for someone to remove.

File: src/client/less/app.less (hunk 1 above)
Comment (kwhitley) [inline src/client/less/app.less:60]: Yeah, the existing section definitions handled the background color then had sub containers for the padding/width restrictions.  This does a simple wrap of the content block to force any injected content (no matter the wrappers) to fall within the white "content" section.

**Comment by kwhitley** [comment]
Cool - would mean the submodules (that inherit its layout) would no longer need to reference it...

:+1:

**Comment by ericclemmons** [comment]
This is related to #22, right?

**Comment by kwhitley** [comment]
@ericclemmons yep my bad!

**Comment by kwhitley** [comment]
Also would like to clean up the views by adding a child layout for docs/ but need to resolve a nested blocks issue w twigjs...

https://github.com/justjohn/twig.js/issues/51

**Comment by kwhitley** [comment]
Can we go ahead and merge this (and possibly amend/revise later with changes)?  Would help make making new features off it much easier...

**Comment by ericclemmons** [comment]
<obama>Yes we can!</obama>

**Pull request merged**